functions.rb

```diff
@@ -0,0 +1,102 @@
+require_relative 'book'
+require_relative 'teacher'
+
+class Functions
+  def initialize(items)
+    @items = items
+  end
+
+  def list_books
+    puts 'List of the books: '
+    puts @items
+    puts @items.class
```

Comment:
Currently, this statement outputting irrelevant information, You can remove it. 👍